runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java

```diff
@@ -110,6 +110,17 @@ default boolean weaknessPrompt()
 		return true;
 	}
 
+	@ConfigItem(
+		position = 8,
+		keyName = "taskCommand",
+		name = "Task Command",
+		description = "Configures whether the quest point command is enabled<br> !task"
```

Comment:
This description should probably be updated

Reply:
Lol, whoops.